pldoc/static/js/ui.js

```diff
@@ -20,6 +20,12 @@ define([
 
         setupHtml: function() {
             $('html').removeClass('no-js');
+            $("#pldoc-nav-elements ul li a").each(function (index, element) {
```

Comment:
NIT: can we keep this selector to class-based attributes only, e.g. ``#pldoc-nav-elements .nav-list-item .pldoc-link``?